packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -113,18 +177,24 @@
 					/>
 				</template>
 			</tera-drilldown-section>
-			<tera-drilldown-preview
-				title="Output Preview"
-				v-model:output="selectedOutputId"
-				@update:output="onUpdateOutput"
-				@update:selection="onUpdateSelection"
-				:options="outputs"
-				is-selectable
-			>
+			<tera-drilldown-preview title="Output Preview">
 				<div>{{ notebookResponse }}</div>
 			</tera-drilldown-preview>
 		</section>
 	</tera-drilldown>
+	<tera-drilldown
```

Comment:
Can you make it smaller, talk to @ngraham76 their is a design for superposed drilldown.

Reply:
https://github.com/DARPA-ASKEM/terarium/assets/33158416/79432896-5724-42a5-b7ef-30dd64b69190

Reply:
Magnificient 🤩